supported/osg-htc/osg-hosted-ce/templates/configmap.yaml

```diff
@@ -73,7 +73,8 @@ data:
 
     SLATE_HOSTED_CE = True
     SCHEDD_ATTRS = $(SCHEDD_ATTRS) SLATE_HOSTED_CE
-
+    OSG_HOSTED_CE = True
```

Comment:
@brianhlin this work?

Reply:
I would add it above the `SCHEDD_ATTRS` line for sanity's sake. HTCondor config should generally allow refs to uninitiated values but it can make it harder to read for a human. That way you can combine your line 75 + line 77 into a single line

Reply:
Updated, look okay?

Reply:
Yup, LGTM

Reply:
Cool, want to give it one more once over? I might get Jeff D to look too and then merge

Reply:
Resolved in 305f7f36661e340717be0683c2e97b7ce481bd0e